src/main/java/nl/cherement/jak/service/UserService.java

```diff
@@ -15,6 +18,16 @@ public UserService(UserRespository repository) {
         super(repository);
     }
 
+    public Hashtable<Long, String> findAllShortened() {
+        Hashtable<Long, String> users = new Hashtable<>();
```

Comment:
Nu ben ik waarschijnlijk aan het mierenneuken maar de benaming is hier een beetje weird. 'users' en 'allUsers'. Klinkt alsof 'users' niet alle users bevat terwijl hij dat wel doet. Beter zou iets zijn van: shortenedUsers en userEntities.